### PR TITLE
Created the new branch for OCL 14.1.0.1 upcoming release

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Created the new branch for OCL 14.1.0.1 upcoming release

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [X] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [X] This PR is for a release, please add this tag: product/OCL-Connector/14.1.0.1

### CODE REVIEW

**Comment by nidhibhammar** [comment]
@drothery-edb, I have added the release notes with the new format. It can be reviewed once it is deployed. After review, I will remove the What's New section.

File: product_docs/docs/jdbc_connector/42.2.19.1/01_release_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: "Release Notes"
+
+---
+The EDB JDBC connector provides connectivity between a Java application and an Advanced Server database.
+
+New features, enhancements, bug fixes, and other changes in the EDB JDBC Connector `42.2.19.1` include:
+
+| Type           | Description                                                                                                                                                                  |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Upstream Merge | Merged with the upstream community driver version `42.2.19`. See [Community JDBC documentation](https://jdbc.postgresql.org/documentation/head/index.html) for details.      |
```
Comment (drothery-edb) [inline product_docs/docs/jdbc_connector/42.2.19.1/01_release_notes.mdx:11]:
```suggestion
| Upstream Merge | Merged with the upstream community driver version `42.2.19`. See the [community JDBC documentation](https://jdbc.postgresql.org/documentation/head/index.html) for details.      |
```

File: product_docs/docs/jdbc_connector/42.2.19.1/01_release_notes.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+| Enhancement    | EDB JDBC Connector now supports GSSAPI encrypted connection.                                                                                                                 |
+
+!!! Note
+    Deprecated support for Java 1.6 and 1.7 versions. Previous version of EDB JDBC Connector v42.2.12.3 will continue to support Java 1.6 and 1.7 versions.
```
Comment (drothery-edb) [inline product_docs/docs/jdbc_connector/42.2.19.1/01_release_notes.mdx:15]:
```suggestion
    Deprecated support for Java 1.6 and 1.7 versions. Previous versions of EDB JDBC Connector v42.2.12.3 continue to support Java 1.6 and 1.7 versions.
```

File: product_docs/docs/jdbc_connector/42.2.24.1/01_release_notes.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+title: "Release Notes"
+
+---
+The EDB JDBC connector provides connectivity between a Java application and an Advanced Server database.
+
+New features, enhancements, bug fixes, and other changes in the EDB JDBC Connector `42.2.24.1` include:
+
+| Type           | Description                                                                                                                                                                  |
+| -------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Upstream Merge | Merged with the upstream community driver version `42.2.24`. See [Community JDBC documentation](https://jdbc.postgresql.org/documentation/head/index.html) for details.      |
```
Comment (drothery-edb) [inline product_docs/docs/jdbc_connector/42.2.24.1/01_release_notes.mdx:11]:
```suggestion
| Upstream Merge | Merged with the upstream community driver version `42.2.24`. See the community [JDBC documentation](https://jdbc.postgresql.org/documentation/head/index.html) for details.      |
```

File: product_docs/docs/net_connector/4.1.6.1/01_release_notes.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
+  - "/edb-docs/d/edb-postgres-net-connector/user-guides/net-guide/4.1.6.1/whats_new.html"
+---
+The EDB .NET Connector provides connectivity between a .NET client application and an Advanced Server database server.
+
+New features, enhancements, bug fixes, and other changes in the EDB .NET Connector `4.1.6.1` include:
+
+| Type           | Description                                                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Upstream Merge | Merged with the upstream Npgsql driver version 4.1.6. For more information about the merge updates, visit <https://www.nuget.org/packages/Npgsql/4.1.6>.        |
```
Comment (drothery-edb) [inline product_docs/docs/net_connector/4.1.6.1/01_release_notes.mdx:14]:
```suggestion
| Upstream Merge | Merged with the upstream Npgsql driver version 4.1.6. For more information about the merge updates, see <https://www.nuget.org/packages/Npgsql/4.1.6>.        |
```

File: product_docs/docs/net_connector/5.0.7.1/01_release_notes.mdx [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+title: "Release Notes"
+
+---
+The EDB .NET Connector provides connectivity between a .NET client application and an Advanced Server database server.
+
+New features, enhancements, bug fixes, and other changes in the EDB .NET Connector `5.0.7.1` include:
+
+| Type           | Description                                                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Upstream Merge | Merged with the upstream Npgsql driver version 5.0.7. For more information about the merge updates, visit <https://www.nuget.org/packages/Npgsql/5.0.7>.        |
```
Comment (drothery-edb) [inline product_docs/docs/net_connector/5.0.7.1/01_release_notes.mdx:11]:
```suggestion
| Upstream Merge | Merged with the upstream Npgsql driver version 5.0.7. For more information about the merge updates, see <https://www.nuget.org/packages/Npgsql/5.0.7>.        |
```

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**